public/app/features/plugins/admin/components/InstallControls/InstallControlsButton.tsx

```diff
@@ -126,24 +126,28 @@ export function InstallControlsButton({
     uninstallTitle = 'Preinstalled plugin. Remove from Grafana config before uninstalling.';
   }
 
+  const renderUninstallControls = (
```

Comment:
Super nit: maybe `uninstallControls` would be a better name, as now it's not a function, but the "actual element".